blocks/header/header.js

```diff
@@ -64,6 +64,6 @@ export default async function decorate(block) {
   });
   nav.prepend(hamburger);
   nav.setAttribute('aria-expanded', 'false');
-
+  await decorateIcons(nav);
```

Comment:
```suggestion
  decorateIcons(nav);
```

As is, the method is not `async`.